terraform/workspace-variables/pen.sh

```diff
@@ -0,0 +1,3 @@
+export TF_VAR_key_vault_name=s121t01-shared-kv-01
+export TF_VAR_key_vault_app_secret_name=REGISTER-APP-SECRETS-STAGING
```

Comment:
should we create a copy of the app secrets for pen test environment?

Reply:
As discussed with @misaka , we can use the same secrets as staging